_posts/2018-04-17-smartroom-api.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Smartroom API
+date: 2018-04-17
+categories:
+    - Project
+    - PHP
+    - Lumen
+    - REST
+    - API
+description: A uniffed URL for accessing the varius parts of CSH's Smart Room Display
+image: https://fthmb.tqn.com/FzECgdCy_W-gXosZw89_-8UDraw=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/PHP-code-58d2d5803df78c51623a6ce2.jpg
+image-sm: https://fthmb.tqn.com/FzECgdCy_W-gXosZw89_-8UDraw=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/PHP-code-58d2d5803df78c51623a6ce2.jpg
+author: Leul Berhane-Meskel
+author-image: 
+author-bio: A 3rd year Computer Science student and an active member of CSH. A Developing Developer.
+author-email: leul@csh.rit.edu
+author-social:
+    github: github.com/LeulBM
+    linkedin: 
+---
+# One API to Rule them All
+In March, the idea of CSH creating an exhibit for ImagineRIT 2018 began floating around. We quickly decided on a Smart Dorm room, a glimps of what could be. It would be a great collaborative
+ project, with a multitude of unique parts creatied by various members, each person being allowed to work and create how they pleased. The only requirement is that the projects are internet 
```

Comment:
> the only requirement is that the projects are

Since the previous sentence was past-tense, I would suggest "was" instead of "is", and "were" instead or "are"